main.js

```diff
@@ -28,41 +28,45 @@ app.use(expressIp().getIpInfoMiddleware);
 app.set('PORT', PORT);
 
 app.get('/', function(req, res) {
-    res.sendFile(__dirname + "/index.html");
+    res.sendFile(__dirname + '/index.html');
     const userInfo = req.ipInfo;
-    console.log(`User connected from ${userInfo.city}, ${userInfo.country}`);
-    console.log(`User details = ${userInfo}`);
+    console.log('User connected from %s, %s', userInfo.city, userInfo.country);
+    console.log('User details = %o', userInfo);
 });
 
 app.get('/input_data', function(req,res) {
-     res.sendFile(__dirname + "/index.html");
+     res.sendFile(__dirname + '/index.html');
 });
 var client = Octokit({
     auth: TOKEN
 });
 
 app.post('/input_data',(req,res) => {
-    if(req.body.handle == "") res.redirect('back'); 
+    if(req.body.handle === '') res.redirect('back');
     client.users
         .getByUsername({
             username: req.body.handle
         })
         .then((userdata) => {
-            var invStatus = client.orgs.createInvitation({
-                org: ORGANIZATION,
-                email: req.body.email
-            });
+            var invStatus = client.orgs
+	        .createInvitation({
+                    org: ORGANIZATION,
+                    email: req.body.email
+                })
+		.catch((error) => {
+                    console.log('Invitation failed %o', error)
+		});
```

Comment:
This could slightly touch #9 @sbvkrishna
Better off to check before perhaps?

Reply:
I have made a pull request for better error handling for Organization part - #21

Reply:
Looks cool!
I'll rebase in case your PR gets merged first.